core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -385,9 +380,7 @@ oppia.controller('SettingsTab', [
       // 'moderator action' path, and implement an option for different actions
       // saying whether emails should be sent for these, or not. At present,
       // we don't expect to send an email when an exploration is unpublicized.
-      explorationRightsService.saveChangeToBackend({
-        is_publicized: false
-      });
+      explorationRightsService.makePublicized(false);
```

Comment:
Interesting. There is unpublishExplorationAsModerator() in SettingsTab. Is that a different code path?